docs/source/pipeline.rst

```diff
@@ -43,8 +43,8 @@ Writeback (WB)
 Multi- and Single-Cycle Instructions
 ------------------------------------
 
-:numref:`Cycle counts per instruction type` shows the cycle count per instruction type. Some instructions have a variable time, this is indicated as a range e.g. 1..32 means
-that the instruction takes a minimum of 1 cycle and a maximum of 32 cycles. The cycle counts assume zero stall on the instruction-side interface
+:numref:`Cycle counts per instruction type` shows the cycle count per instruction type. Some instructions have a variable time, this is indicated as a range e.g. 3..35 means
+that the instruction takes a minimum of 3 cycles and a maximum of 35 cycles. The cycle counts assume zero stall on the instruction-side interface
```

Comment:
why minimum is 3 and max 35?

Reply:
because that's the only example which is used in the text which uses that notation - so it's better to explain the only real usage